File: tests/human/x/clj/break_continue.clj
(ns break-continue)

(def numbers [1 2 3 4 5 6 7 8 9])

(loop [xs numbers]
  (when-let [n (first xs)]
    (cond
      (even? n) (recur (rest xs))
      (> n 7) nil
      :else (do (println "odd number:" n)
                (recur (rest xs))))))

File: tests/aster/x/lua/break_continue.lua
numbers = {1, 2, 3, 4, 5, 6, 7, 8, 9};
for _, n in ipairs(numbers) do
  if ((n % 2) == 0) then
    goto __cont_1;
  end;
  if (n > 7) then
    break;
  end;
  print(string.format("odd number: %s", n));
  ::__cont_1::;
end;
